### PR TITLE
Configurable ascii images

### PR DESCRIPTION
**Pull request by klipach**
Some people have bad sense of humor, so i added some configs for ascii images. Now you can change them or even disable.

### CODE REVIEW

File: src/GrumPHP/Console/Helper/PathsHelper.php [hunk 1]
```diff
@@ -80,9 +80,16 @@ public function getAsciiPath()
      */
     public function getAsciiContent($resource)
     {
-        $file = $this->getAsciiPath() . $resource . '.txt';
-        if (!$this->fileSystem->exists($file)) {
-            throw new RuntimeException(sprintf('ASCII file %s could not be found.', $resource));
```
Comment (klipach) [inline src/GrumPHP/Console/Helper/PathsHelper.php]: I think it is better to continue script execution even if there is no ascii image

File: src/GrumPHP/Configuration/GrumPHP.php [hunk 2]
```diff
@@ -56,4 +56,25 @@ public function getTaskConfig($taskName = null)
...
     }
+
+    /**
+     * Get ascii content path from grumphp.yml file
+     *
+     * @param $resource
+     *
+     * @return mixed
+     */
+    public function getAsciiContentPath($resource)
```
Comment (veewee) [inline src/GrumPHP/Configuration/GrumPHP.php]: Can you provide a spec for this method?

**Comment by veewee** [comment]
Hello @klipach,

Thanks again for contributing! This sure looks like a handy feature.
A little sidenode: We are planning to create a mascot for GrumPHP. Afterwords the current success and error message will be replaced by this mascot. We are trying to keep GrumPHP neutral yet fun to work with :)

Can you please add a spec for the Configuration method and fix CS.
I will try to make some time during the weekend to merge, test and release a new version.

**Comment by veewee** [comment]
Hi @klipach ,
I fixed the code and added some tests.
Also the default configuration is made explicit. This way the services.yml file is the only source of truth.
Now it is also possible to rename the ASCII files without changing the code.

Thanks!

**Comment by klipach** [comment]
Great! Sorry was off for several days was not able to fix comments.

**Comment by veewee** [comment]
No problem! :)

**Pull request merged**